src/Converter/Feature2Texture.js

```diff
@@ -166,25 +171,38 @@ export default {
             ctx.imageSmoothingEnabled = false;
             ctx.lineJoin = 'round';
 
-            const ex = collection.crs == extent.crs ? extent : extent.as(collection.crs, _extent);
-            const t = collection.translation;
-            const s = collection.scale;
-            extentTransformed.transformedCopy(t, s, ex);
+            // transform extent to feature projection
+            extent.as(collection.crs, featureExtent);
+            // transform extent to local system
+            featureExtent.applyMatrix4(collection.matrixWorldInverse);
+
+            // compute matrix transformation `world2texture` to convert coordinates to texture coordinates
+            if (collection.isInverted) {
+                worldTextureOrigin.set(extent.west, extent.north, 0);
+                canvasSize.set(ctx.canvas.width, -ctx.canvas.height, 1.0);
+            } else {
+                worldTextureOrigin.set(extent.west, extent.south, 0);
+                canvasSize.set(ctx.canvas.width, ctx.canvas.height, 1.0);
+            }
+
+            world2texture.compose(worldTextureOrigin, quaternion, dimension.divide(canvasSize)).invert();
```

Comment:
Should the `invert()` be removed, you could try :
```js
const scale = canvasSize.divide(dimension);
world2texture.compose(
    worldTextureOrigin.multiply(scale).negate(),
    quaternion,
    scale,
);
```
Assumed the quaternion is allways neutral (i.e. no rotation), it gives the same result than : 
```js
world2texture.compose(worldTextureOrigin, quaternion, dimension.divide(canvasSize)).invert()
```

Reply:
yes, i changed